packages/api/bin/es.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+
+/* eslint-disable no-console */
+/* eslint-disable no-param-reassign */
+
+const { Search, defaultIndexAlias } = require('../es/search');
+const mappings = require('../models/mappings.json');
+
+/**
+ * Complete the Elasticsearch reindex by swapping the alias from source index to
+ * dest index so that dest index will now be the index used by Cumulus
+ *
+ * @param {string} host - Elasticsearch host
+ * @param {string} sourceIndex - name of the index to swap from
+ * @param {string} destIndex - name of the index to swap to
+ * @param {string} aliasName - alias name that the instance is using, defaults to cumulus-alias
+ * @param {boolean} deleteSource - true to delete the source index
+ * @returns {undefined} - none
+ */
+async function completeReindex(
+  host,
+  sourceIndex,
+  destIndex,
+  aliasName = defaultIndexAlias,
+  deleteSource = false
+) {
+  const esClient = await Search.es(host);
+
+  if (sourceIndex === null || destIndex === null) {
+    // eslint-disable-next-line max-len
+    throw new Error('Please explicity specify a source and destination index.');
+  }
+
+  if (sourceIndex === destIndex) {
+    throw new Error('The source index cannot be the same as the destination index.');
+  }
+
+  const sourceExists = await esClient.indices.exists({ index: sourceIndex });
+
+  if (!sourceExists) {
+    throw new Error(`Source index ${sourceIndex} does not exist.`);
+  }
+
+  const destExists = await esClient.indices.exists({ index: destIndex });
+
+  if (!destExists) {
+    throw new Error(`Destination index ${destIndex} does not exist.`);
+  }
+
+  await esClient.indices.updateAliases({
+    body: {
+      actions: [
+        { remove: { index: sourceIndex, alias: aliasName } },
+        { add: { index: destIndex, alias: aliasName } }
+      ]
+    }
+  }).then(() => {
+    // eslint-disable-next-line max-len
+    console.log(`Removed alias ${aliasName} from index ${sourceIndex} and added alias to ${destIndex}`);
+  }, (err) => {
+    // eslint-disable-next-line max-len
+    throw new Error(`Error removing alias ${aliasName} from index ${sourceIndex} and adding alias to ${destIndex}: ${err}`);
+  });
+
+  if (deleteSource) {
+    await esClient.indices.delete({ index: sourceIndex });
+    console.log(`Deleted index ${sourceIndex}`);
+  }
+}
+
+/**
+ * Get the Elasticsearch reindexing status by listing the tasks that are of
+ * type reindex
+ *
+ * @param {string} host - Elasticsearch host
+ * @returns {Object} - details of the tasks from Elasticsearch
+ */
+async function getStatus(host) {
+  const esClient = await Search.es(host);
+
+  const tasks = await esClient.tasks.list({ actions: ['*reindex'] });
```

Comment:
Not 100% on this but I don't think we need an `await` both here and in line 51 of the cli. I think you can:

```javascript
function getStatus(host) {
  const esClient = await Search.es(host);
  return esClient.tasks.list({ actions: ['*reindex'] }).promise();
};
```
And leave the `await`ing to the cli.

My understanding of `await` is it's really only necessary within a function when you need to wait for a promise to be resolved and do something with the result in the same function. Within `getStatus` we're not doing anything with the result so we don't need to `await` for it.

cc @yjpa7145 to correct me if I'm wrong.